app/src/main/java/org/hackillinois/android/view/home/CountdownManager.kt

```diff
@@ -9,17 +9,17 @@ class CountdownManager(val listener: CountDownListener) {
 
     private val eventStartTime: Calendar = Calendar.getInstance().apply {
         timeZone = TimeZone.getTimeZone("America/Chicago")
-        set(2019, Calendar.FEBRUARY, 22, 16, 0, 0)
+        timeInMillis = 1582927200000
```

Comment:
Don't forget to use https://api.hackillinois.org/upload/blobstore/times/
I assume you're going to do that in another PR

Reply:
Yep, that'll come soon.